.github/actions/on_device_tests/action.yaml

```diff
@@ -89,3 +92,22 @@ runs:
           --label author-${GITHUB_PR_HEAD_USER_LOGIN:-$GITHUB_COMMIT_AUTHOR_USERNAME} \
           --label author_id-${GITHUB_PR_HEAD_USER_ID:-$GITHUB_COMMIT_AUTHOR_EMAIL}
       shell: bash
+    - name: Download ${{ matrix.platform }} Test Results
+      if: env.TEST_TYPE == 'unit_test'
+      run: |
+        set -eux
+        gsutil -d cp "${GCS_RESULTS_PATH}" results.zip
+        if [[ $? -ne 0 ]]; then
```

Comment:
This check will be removed?

Reply:
I pushed the branch to my fork instead of here. Should be updated now.